src/gui/transferlistmodel.cpp

```diff
@@ -55,6 +55,76 @@ static QIcon getErrorIcon();
 
 static bool isDarkTheme();
 
+TransferListTextColors::TransferListTextColors()
+    : m_unknown{getColorByState(BitTorrent::TorrentState::Unknown)}
```

Comment:
you need to add a space `m_unknown {...}`

Reply:
done?

Reply:
you misunderstood. Look for examples in session.cpp. Requirement for spaces are different for () and {}.

Reply:
now?